packages/react-events/src/Hover.js

```diff
@@ -265,16 +277,26 @@ const HoverResponder = {
               dispatchHoverStartEvents(event, context, props, state);
               state.isInHitSlop = false;
             }
-          } else if (
-            state.isHovered &&
-            context.isPositionWithinTouchHitTarget(
-              target.ownerDocument,
-              (nativeEvent: any).x,
-              (nativeEvent: any).y,
-            )
-          ) {
-            dispatchHoverEndEvents(event, context, props, state);
-            state.isInHitSlop = true;
+          } else if (state.isHovered) {
+            if (
+              context.isPositionWithinTouchHitTarget(
+                target.ownerDocument,
+                (nativeEvent: any).x,
+                (nativeEvent: any).y,
+              )
+            ) {
+              dispatchHoverEndEvents(event, context, props, state);
+              state.isInHitSlop = true;
+            } else {
+              if (props.onHoverMove) {
+                const syntheticEvent = createHoverEvent(
+                  'hovermove',
+                  event.target,
+                  props.onHoverMove,
+                );
+                context.dispatchEvent(syntheticEvent, {discrete: true});
```

Comment:
Hover move shouldn't be discrete. Come to think of it, is press move discrete? We should also ensure that is not either.